lib/tapioca/dsl/helpers/active_record_column_type_helper.rb

```diff
@@ -35,6 +35,8 @@ def type_for(column_name)
               "::Float"
             when ActiveRecord::Type::Boolean
               "T::Boolean"
+            when ActiveRecord::ConnectionAdapters::PostgreSQL::OID::Hstore
```

Comment:
We need to be smarter and reference this constant only when the host application is using Postgres. Other constants on this file are loaded by default when we load ActiveRecord and don't need this treatment https://github.com/Shopify/tapioca/blob/3a2a8d150cd6501e78251c25098b530df119ac84/lib/tapioca/dsl/compilers/active_record_columns.rb#L5

Other than that I'm okay with supporting database specific columns but if there are more of them we can look into a more extensible approach. Such as having a different helper file for Postgres.

Reply:
I've updated this to check that `ActiveRecord::ConnectionAdapters::PostgreSQL` is defined, and also extended it to correctly type postgres Array columns.